google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerImpl.java

```diff
@@ -216,15 +244,22 @@ public void cancelled(Context context) {
   Session createSession(final DatabaseId db) throws SpannerException {
     final Map<SpannerRpc.Option, ?> options =
         optionMap(SessionOption.channelHint(random.nextLong()));
-    com.google.spanner.v1.Session session =
-        runWithRetries(
-            new Callable<com.google.spanner.v1.Session>() {
-              @Override
-              public com.google.spanner.v1.Session call() throws Exception {
-                return rpc.createSession(db.getName(), getOptions().getSessionLabels(), options);
-              }
-            });
-    return new SessionImpl(session.getName(), options);
+    Span span = tracer.spanBuilder(CREATE_SESSION).startSpan();
+    try (Scope s = tracer.withSpan(span)) {
+      com.google.spanner.v1.Session session =
+          runWithRetries(
+              new Callable<com.google.spanner.v1.Session>() {
```

Comment:
FYI: You can add a TODO to change to the new API that we add in 0.10.0 to wrap callable/runnable see:
https://github.com/census-instrumentation/opencensus-java/pull/778